mod-pubsub-server/src/main/java/org/folio/representation/User.java

```diff
@@ -0,0 +1,82 @@
+package org.folio.representation;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+
+public class User {
```

Comment:
I believe lombok has been introduced in other places to reduce some of this boilerplate code. Could it be introduced here as well?

Reply:
I'm fine to introduce lombok here as well. unless the team has any objections.

Reply:
@marcjohnson-kint, I'll leave to the team to add this dependency. Having it just for a single class not give much benefit.